libraries/src/HTML/Helpers/UiTab.php

```diff
@@ -54,12 +54,15 @@ public static function startTabSet($selector = 'myTab', $params = array())
 			static::$loaded[__METHOD__][$selector]['active'] = $opt['active'];
 		}
 
-		// @TODO echo the recall attribute correctly, now it's hardcoded!!!
-		$recall = !isset($params['recall']) ? '' : 'recall';
-
 		$orientation = isset($params['orientation']) ? $params['orientation'] : 'horizontal';
+		$recall = isset($params['recall']) ? 'recall' : '';
+		$breakpoint  = isset($params['breakpoint']) ? 'breakpoint="' . $params['breakpoint'] . '"' : '';
+
+		if (!isset($params['breakpoint']) && $breakpoint === '') {
```

Comment:
```suggestion
		if (!isset($params['breakpoint']) && $breakpoint === '')
		{
```
Fix PHP CS error in Drone.